anet.yml

```diff
@@ -280,6 +280,7 @@ dictionary:
               label: "No"
         numberTrained:
           type: number
+          typeError: Number trained must be a number
```

Comment:
shouldn't this message be "Number trained must be an integer" as the validation is for an integer?